Monika After Story/game/event-handler.rpy

```diff
@@ -2095,7 +2102,11 @@ label prompt_menu:
         talk_menu.append(("Hey, [m_name]...", "prompt"))
         if len(repeatable_events)>0:
             talk_menu.append(("Repeat conversation", "repeat"))
-        talk_menu.append(("I love you!", "love"))
+        if _mas_getAffection() > -50:
+            if persistent._mas_last_monika_ily is not None and (datetime.datetime.now() - persistent._mas_last_monika_ily) <= datetime.timedelta(0,10):
```

Comment:
seems to me that checking if an amount of time has passed since an ily would be useful in other places, too. Can you turn this conditional check into a function (probably put in definitions)? And have a timedelta as the argument.